.github/workflows/create-xray-test-plan-and-test-execution.yml

```diff
@@ -13,14 +13,30 @@ on:
         required: true
         type: string
     outputs:
-      test_plan_key:
-        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_plan_key }}
-      test_plan_id:
-        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_plan_id }}
-      test_execution_key:
-        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_execution_key }}
-      test_execution_id:
-        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_execution_id }}
+      test_plan_key_alma8:
+        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_plan_key_alma8 }}
+      test_plan_id_alma8:
+        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_plan_id_alma8 }}
+      test_execution_key_alma8:
+        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_execution_key_alma8 }}
+      test_execution_id_alma8:
+        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_execution_id_alma8 }}
+      test_plan_key_alma9:
+        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_plan_key_alma9 }}
+      test_plan_id_alma9:
+        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_plan_id_alma9 }}
+      test_execution_key_alma9:
+        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_execution_key_alma9 }}
+      test_execution_id_alma9:
+        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_execution_id_alma9 }}
+      test_plan_key_bullseye:
+        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_plan_key_bullseye }}
+      test_plan_id_bullseye:
+        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_plan_id_bullseye }}
+      test_execution_key_bullseye:
+        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_execution_key_bullseye }}
+      test_execution_id_bullseye:
+        value: ${{ jobs.create-xray-test-plan-and-test-execution.outputs.test_execution_id_bullseye }}
```

Comment:
note for reviewers : 
it is currently not possible to generate dynamic output with composite action : using following workaround => https://github.com/orgs/community/discussions/26639#discussioncomment-3949595
it is only possible with js action, so we'll maybe move to later to javascript action